src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -15,6 +15,21 @@ api_platform:
         SM\SMException: 422
         Sylius\Bundle\ApiBundle\Exception\CannotRemoveCurrentlyLoggedInUser: 422
         Sylius\Bundle\ApiBundle\Exception\ShippingMethodCannotBeRemoved: 422
+# Configurations for Swagger still doesn't work properly
+    oauth:
+        enabled: true
+        clientId: '5sxomk2lm680w80w00s0kssc4so4cccg0ksokks4csc88skgo0'
```

Comment:
This values are hardcoded, however, should they be fetched from the results of the command you've written above? Is it the main problem, why OAuth is not working properly?

Reply:
In next iteration it should be like global variables? for example`%oauth_client_id%` that would be assigned after running the command, I think problem was not here